Pod/Classes/LSTracer.m

```diff
@@ -24,7 +24,7 @@
 @implementation LSTracer {
     NSString* m_accessToken;
     UInt64 m_runtimeGuid;
-    NSDictionary<NSString*, NSString*>* m_tracerJSON;
+    NSDictionary<NSString*, NSObject *>* m_tracerJSON;
```

Comment:
@bensigelman if you look in the commit message body, we're passing  a dictionary as a value below :)

https://github.com/lightstep/lightstep-tracer-objc/pull/30/files#diff-dcd31561a7bbcafcbdeb70ddd1ba0b08R59